backend/src/packages/chaiNNer_pytorch/settings.py

```diff
@@ -61,6 +72,12 @@ class PyTorchSettings:
     use_fp16: bool
     gpu_index: int
 
+    # PyTorch 2.0 on macOS does not support FP16 when using CPU
+    def __post_init__(self):
+        if self.use_cpu and self.use_fp16 and is_arm_mac:
```

Comment:
This actually would apply to anything, not just arm macs
```suggestion
        if self.use_cpu and self.use_fp16
```

Reply:
done